.github/workflows/workflow.yaml

```diff
@@ -46,7 +46,7 @@ jobs:
     - name: Test with pytest
       run: |
         pip install --upgrade pytest
-        pytest --run-cfchecks --cov=./ --cov-report=xml
+        pytest -s --run-cfchecks --cov=./ --cov-report=xml
```

Comment:
Do we still need `-s` in the pytest checks or was this just needed for checking the issue this PR is fixing?

Reply:
Yeah there was an issue with the capture of output so I added that, I've removed it now and everything works.

Reply:
Great, makes sense.